com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -493,7 +493,10 @@ static void RenderCameraStack(ScriptableRenderContext context, Camera baseCamera
 
 #if ENABLE_VR && ENABLE_XR_MODULE
             var xrActive = false;
-            var xrPasses = m_XRSystem.SetupFrame(baseCamera, baseCameraAdditionalData.allowXRRendering);
+            var xrRendering = true;
```

Comment:
should this be false by default ?

Reply:
I think this should be true by default so we treat all cameras as XR cameras by default. The reason for doing that is because when converting a project from built-in to URP, the cameraAdditionalData would be null and if we set xrRendering to false by default, the XR will be disabled for those cameras.